src/main/resources/liquibase/crop_changelog/14_2_0.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="gelli" id="v14.2.0-1">
+        <preConditions onFail="MARK_RAN">
+            <and>
+                <sqlCheck expectedResult="1">
+                    SELECT CASE WHEN (SELECT COUNT(*) FROM germplsm WHERE methn = 0) > 0 THEN 1 ELSE 0 END;
+                </sqlCheck>
+                <sqlCheck expectedResult="2">
+                    SELECT COUNT(*) FROM methods WHERE mid IN (1,31);
```

Comment:
Just noting that I think it is safe to use method IDs 1 and 31 as it's included in common [methods](https://github.com/IntegratedBreedingPlatform/DBScripts/blob/master/merged/common/20_Data_Common_Methods.sql) seeded for all crops